Dockerfile.rhel7

```diff
@@ -1,7 +1,7 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
 WORKDIR /go/src/github.com/openshift/service-ca-operator
 COPY . .
-RUN go build -o service-ca-operator ./cmd/service-ca-operator
+RUN GODEBUG=tls13=1 go build -o service-ca-operator ./cmd/service-ca-operator
```

Comment:
wat?

Reply:
see the note above from @damemi? with a bump in library-go, this is required or else when building the image you'll see 
```
STEP 4: RUN go build -o service-ca-operator ./cmd/service-ca-operator
# github.com/openshift/service-ca-operator/vendor/github.com/openshift/library-go/pkg/crypto
vendor/github.com/openshift/library-go/pkg/crypto/crypto.go:39:18: undefined: tls.VersionTLS13
# github.com/openshift/service-ca-operator/vendor/k8s.io/component-base/cli/flag
vendor/k8s.io/component-base/cli/flag/ciphersuites_flag.go:80:18: undefined: tls.VersionTLS13
error building at STEP "RUN go build -o service-ca-operator ./cmd/service-ca-operator": error while running runtime: exit status 2
```
weird, i agree.